test/Hydration-test.js

```diff
@@ -103,9 +102,9 @@ describe('Hydration', function() {
   });
 
   it.skip('can calculate how many fluid ounces a user consumed on each day for the last 7 days', function() {
-    expect(userOneHydration.calculateOuncesConsumedOnDate('2019/06/17')).to.equal(96);
-    expect(userOneHydration.calculateOuncesConsumedOnDate('2019/06/16')).to.equal(69);
+    expect(userOneHydration.calculateWeeklyConsumption('2019/06/21')).to.deep.equal([37, 69, 96, 100, 20, 31, 91]);
+    expect(userOneHydration.calculateWeeklyConsumption('2019/06/22')).to.deep.equal([69, 96, 100, 20, 31, 91, 55]);
+    expect(userTwoHydration.calculateWeeklyConsumption('2019/06/22')).to.be.undefined;
```

Comment:
we could decide that we want to return a shorter array (rather than undefined) for user with less than seven days of hydration data

Reply:
this could prolly be resolved with a single `if` statement. Like ```if (arr.length > 6) { return arr.slice(0, 6);} else {return arr}```